awxkit/awxkit/cli/resource.py

```diff
@@ -44,6 +49,56 @@
     (v, k) for k, v in DEPRECATED_RESOURCES.items()
 )
 
+EXPORTABLE_RESOURCES = [
+    'users',
+    'organizations',
+    'teams',
+    'credential_types',
+    'credentials',
+    # 'notification_templates',
+    # 'projects',
+    # 'inventory',
+    # 'job_templates',
+    # 'workflow_job_templates',
+]
+
+NATURAL_KEYS = {
+    'user': ('username',),
+    'organization': ('name',),
+    'team': ('organization', 'name'),
+    'credential_type': ('name', 'kind'),
+    'credential': ('organization', 'name', 'credential_type'),
+    'notification_template': ('organization', 'name'),
+    'project': ('organization', 'name'),
+    'inventory': ('organization', 'name'),
+    'job_template': ('name',),
+    'workflow_job_template': ('organization', 'name'),
+
+    # related resources
+    'role': ('name', ':content_object'),
```

Comment:
Could you produce these from the named URL?

Reply:
No, I don't think so.  I'm deliberately packing all of the (potentially nested) information needed to reconstruct the link into the data structure, rather than relying on ad hoc program logic to figure it out from minimal information.  The results in the export look something like:

```json
    {
         "type": "role",
         "name": "Admin",
         "content_object": {
              "type": "credential",
              "organization": null,
              "name": "Demo Credential",
              "credential_type": {
                   "type": "credential_type",
                   "name": "Machine",
                   "kind": "ssh"
              }
         }
    },
```